src/components/CardImage/CardImage.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom"
+
+interface CardImageProps {
+  title: string
+  path: string
+  imgUrl: string
+}
+
+export const CardImage = (props: CardImageProps) => {
```

Comment:
Ogólnie ja bym te dwa komponenty połączył w jeden i niech działa tak że, trzeba podać kolor zawsze, ale zdjęcie może być opcjonalne. Jak zdjęcie jest podane to będzie wyświetlane zamiast tego koloru. Mamy wtedy jeden komponent i jak będziemy mapować elementy to nie musimy się zastanawiać które komponenty wykorzystać.

Reply:
Zrobione, aczkolwiek kolor też dałem opcjonalny, ponieważ, nie widzę sensu aby go dawać, kiedy jest zdjęcie.